libopeniscsiusr/default.h

```diff
@@ -57,6 +57,9 @@
 /* session reopen max retries */
 #define	DEF_SESSION_REOPEN_MAX		0
 
+/* session reopen log frequency */
```

Comment:
could you add something to the comment like "number of reopens per message", since "frequency" could be ambiguously interpreted?

Reply:
Done